### PR TITLE
sieve of eratosthenes python code

### PR DESCRIPTION
**Pull request by udaykiranreddykondreddy**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

Fixes #{ISSUE_NUMBER}

**By submitting this pull request I confirm I've read and complied with the
below declarations.**

- [x] I have read the [Contribution guidelines](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [commit guidelines](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md#write-good-commit-messages) for this project.
- [x] My code follows the [standard code structure](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md#samples-code).
- [x] This pull request has a descriptive title. For example, `{Tag}: Add {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.
- [x] This pull request shall only be reviewed and merged once all the checks passes. No maintainer or supporter shall be obliged to review it before this condition is met.
- [x] I have mentioned the issue number correctly (with hyperlink) in this pull request description.

After you submit your pull request, **DO NOT** click the 'Update Branch' button.

### CODE REVIEW

File: sieve_of_Eratosthenes.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
+def sieve_of_eratosthenes(n):
+
```
Comment (sangamcse) [inline sieve_of_Eratosthenes.py]: Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `all.pyjava`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp5np8ml1e/sieve_of_Eratosthenes.py
+++ b/tmp/tmp5np8ml1e/sieve_of_Eratosthenes.py
@@ -1,5 +1,5 @@
 def sieve_of_eratosthenes(n):
-    
+
     prime = [True for i in range(n+1)]
     p = 2
     while (p * p <= n):
```

Comment (sangamcse) [inline sieve_of_Eratosthenes.py]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp5np8ml1e/sieve_of_Eratosthenes.py
+++ b/tmp/tmp5np8ml1e/sieve_of_Eratosthenes.py
@@ -1,5 +1,5 @@
 def sieve_of_eratosthenes(n):
-    
+
     prime = [True for i in range(n+1)]
     p = 2
     while (p * p <= n):
```

File: sieve_of_Eratosthenes.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+            for i in range(p * 2, n+1, p):
+                prime[i] = False
+        p += 1
+    # Print all prime numbers
+    for p in range(2, n):
+        if prime[p]:
+            print(p)
+
+
+if __name__=='__main__':
```
Comment (sangamcse) [inline sieve_of_Eratosthenes.py]: The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp5np8ml1e/sieve_of_Eratosthenes.py
+++ b/tmp/tmp5np8ml1e/sieve_of_Eratosthenes.py
@@ -15,7 +15,7 @@
             print(p)
 
 
-if __name__=='__main__':
+if __name__ == '__main__':
     n = 30
     print("Following are the prime numbers smaller")
     print("than or equal to", n)
```

File: sieve_of_Eratosthenes.py (hunk 1 above)
Comment (sangamcse) [inline sieve_of_Eratosthenes.py]: W293 blank line contains whitespace

*Origin: PycodestyleBear (W293), Section: `all.autopep8`.*

File: sieve_of_Eratosthenes.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+def sieve_of_eratosthenes(n):
+    
+    prime = [True for i in range(n+1)]
+    p = 2
+    while (p * p <= n):
+        # If prime[p] is not changed, then it is a prime
+        if (prime[p] == True):
```
Comment (sangamcse) [inline sieve_of_Eratosthenes.py]: E712 comparison to True should be 'if cond is True:' or 'if cond:'

*Origin: PycodestyleBear (E712), Section: `all.autopep8`.*

File: sieve_of_Eratosthenes.py (hunk 2 above)
Comment (sangamcse) [inline sieve_of_Eratosthenes.py]: E225 missing whitespace around operator

*Origin: PycodestyleBear (E225), Section: `all.autopep8`.*

Comment (udaykiranreddykondreddy) [reply sieve_of_Eratosthenes.py]: done

File: sieve_of_Eratosthenes.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        p += 1
+    # Print all prime numbers
+    for p in range(2, n):
+        if prime[p]:
+            print(p)
+
+
+if __name__ == '__main__':
+    n = 30
+    print("Following are the prime numbers smaller than or equal to {}: {}".format(n, sieve_of_eratosthenes(n))
```
Comment (sangamcse) [inline sieve_of_Eratosthenes.py]: You need to break line. We accept 80 characters per line. so make it like
```py
print("Following are the prime numbers smaller than "
      "or equal to {}: {}".format(n, sieve_of_eratosthenes(n))
```

**Comment by sangamcse** [comment]
Comment on 0e7d971a8be0286044e30e4f2308fd315583153d, file sieve_of_Eratosthenes/sieve_of_Eratosthenes.py, line 1.

The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp9rxd4ynf/sieve_of_Eratosthenes/sieve_of_Eratosthenes.py
+++ b/tmp/tmp9rxd4ynf/sieve_of_Eratosthenes/sieve_of_Eratosthenes.py
@@ -1 +0,0 @@
-
```

**Comment by sangamcse** [comment]
Comment on 0e7d971a8be0286044e30e4f2308fd315583153d, file sieve_of_Eratosthenes/sieve_of_Eratosthenes.py, line 1.

W391 blank line at end of file

*Origin: PycodestyleBear (W391), Section: `all.autopep8`.*

**Pull request closed without merging**